CHANGELOG.asciidoc

```diff
@@ -305,6 +305,10 @@ https://github.com/elastic/beats/compare/v6.4.0...v6.5.0[View commits]
 - Add Beats Central Management {pull}8559[8559]
 - Report configured queue type. {pull}8091[8091]
 - Enable `host` and `cloud` metadata processors by default. {pull}8596[8596]
+- Dissect will now flag event on parsing error. {pull}8751[8751]
```

Comment:
something went wrong here?

Reply:
Weird, this happened automatically in my rebase. Hmmm